app/src/androidTest/java/com/aletheiaware/perspective/android/GameActivityInstrumentedTest.java

```diff
@@ -259,6 +259,11 @@ public void screenshotWorld1Puzzle6() throws Exception {
         captureScreenshot(PerspectiveAndroidUtils.WORLD_ONE, 6);
     }
 
+    @Test
+    public void screenshotWorld1Puzzle7() throws Exception {
+        captureScreenshot(PerspectiveAndroidUtils.WORLD_ONE, 7);
```

Comment:
For The Future (FTF): I'm guessing you manually review the screen captures, any possible way to "validate" these at test time?

Reply:
The puzzles themselves are validated before being committed to PerspectivePuzzle. These screenshot tests are review manually for consistency before the puzzles are play tested.